main/reference/zoe-api/zoe-data-types.md

```diff
@@ -83,11 +54,34 @@ is exclusively yours and no one else has access to it.
 
 ## Keyword
 
-An ASCII identifier string that must begin with an upper case letter
-in order to avoid collisions with JavaScript properties such as `toString`.
+A **Keyword** is a unique identifier string within a contract for tying together its
+**proposals**, **payments**, and **payouts**.
+It must be a valid ASCII [identifier](https://developer.mozilla.org/en-US/docs/Glossary/Identifier)
+and start with an upper case letter in order to avoid collisions with JavaScript properties
+such as `toString` when used as a property name in a record.
 (For more detail, see [Why do Zoe keywords have to start with a capital letter? #8241](https://github.com/Agoric/agoric-sdk/discussions/8241).)
 `NaN` and `Infinity` are also not allowed as keywords.
 
+<a name="amountkeywordrecord"></a>
```

Comment:
Doesn't that HTML tag have to surround something to have an effect? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a says to use `id` instead.

stackoverflow thinks it's not okay for it to be empty.

Reply:
I’ve found that this works, but the browser is not obliged to scroll farther than necessary to put the top of this marker above the fold. Capturing content obliges the browser to scroll to show as much of the content as fits.

Reply:
`id` is definitely the correct attribute (_my muscle memory here is very old and outdated_), but inner content is not necessary to establish what I know of as an "anchor link". This particular technique of [using an empty `a` element before a Markdown section heading](https://gist.github.com/rachelhyman/b1f109155c9dafffe618?permalink_comment_id=3483183#gistcomment-3483183) allows external links using an old target to still land in approximately the right place without suffering automatic slug generation issues that cause otherwise preferable Markdown like `## KeywordRecord <a id="amountkeywordrecord"></a>` to render incorrectly in many implementations, e.g.:
```diff
--- expected
+++ actual
-<h2 id="keywordrecord">KeywordRecord <a id="amountkeywordrecord"></a></h2>
+<h2 id="keywordrecord-a-id-amountkeywordrecord">KeywordRecord <a id="amountkeywordrecord"></a></h2>
```